File: Training1/G_sms.cpp
#include <bits/stdc++.h>

using namespace std;

int count(string line, unordered_map<char,int> umap){
    int count = 0;
    for (int i=0; i<line.length(); i++){
        char temp = line[i];
        count += umap[temp];
    }

    return count;
}

int main(){
    unordered_map<char, int> umap;
    umap['a'] = 1; umap['b']=2; umap['c']=3;
    umap['d'] = 1; umap['e']=2; umap['f']=3;

    umap['g'] = 1; umap['h']=2; umap['i']=3;
    umap['j'] = 1; umap['k']=2; umap['l']=3;

    umap['m'] = 1; umap['n']=2; umap['o']=3;
    umap['p'] = 1; umap['q']=2; umap['r']=3; umap['s']=4;

    umap['t'] = 1; umap['u']=2; umap['v']=3;
    umap['w'] = 1; umap['x']=2; umap['y']=3; umap['z']=4;
    umap[' ']=1;

    int t;
    vector<string> lines;
    string line;
    cin >> t;
    for (int i=0; i<t; i++){
        getline(cin, line);
        cin.ignore(2,'\n');
        lines.push_back(line);
    }
    for (int i=0; i<t; i++){
        cout << "--" << endl;
        cout << lines[i];
    }

    return 0;
}

File: Training1/G_sms.py

count_dict = {'a': 1, 'b':2, 'c':3,
'd': 1, 'e':2, 'f':3,
'g': 1, 'h':2, 'i':3,
'j': 1, 'k':2, 'l':3,
'm': 1, 'n':2, 'o':3,
'p': 1, 'q':2, 'r':3, 's':4,
't': 1, 'u':2, 'v':3,
'w': 1, 'x':2, 'y':3, 'z':4, ' ':1}

n = int(input())
lines = []
for i in range(n):
    lines.append(input())

for i in range(n):
    line = lines[i]
    count = 0
    for letter in line:
        count += count_dict[letter]
    
    print("Case #{}: {}".format(i+1, count))